api/pyproject.toml

```diff
@@ -73,12 +73,13 @@ spectree = "1.2.1"
 # (https://docs.sqlalchemy.org/en/20/changelog/changelog_14.html#change-e67bfa1efbe52ae40aa842124bc40c51).
 sqlalchemy = { version = "1.4.54", extras = ["mypy"] }
 time-machine = "^2.16.0"
-weasyprint = "62.1"
+weasyprint = "63.1"
 werkzeug = "2.0.3"
 wtforms = "3.2.1"
 wtforms-sqlalchemy = "0.4.2"
 xlsxwriter = "^3.2.0"
 zeep = "4.3.1"
+html5lib = "^1.1"
```

Comment:
(détail) c'est trié par ordre alphabétique. Poetry le rajoute à la fin, je garderais bien la beauté de l'homogénéité :)